doorbell/doorbell_backend/src/detection.py

```diff
@@ -1,23 +1,22 @@
+# TODO: prune imports to specific functions from libraries to reduce .exe size
 import cv2
 import sys
-import socket  # maybe use this for sending files, but if there's a library that does HTTPS connections use that instead
 from pathlib import Path
 import time
 from datetime import datetime
+from datetime import timezone
+from datetime import timedelta
 import os
-import requests
-
-
-def cleanLogs(log_path):
-    cache_list = os.listdir(log_path)
-    current_date_time = datetime.now().strftime("%m-%d-%Y,%H-%M-%S")
 
+import connect
+import logupdate
 
 if __name__ == "__main__":
     casc_path = (
-        str(Path(__file__).parent.parent) + r"\data\haarcascade_frontalface_default.xml"
+        str(Path(__file__).parent.parent)
+        + r"\\data\\haarcascade_frontalface_default.xml"
```

Comment:
Maybe we can move regex strings to separate variables to make this easier to read (maybe a `constants` file where you store all these strings.

Reply:
That's a good idea. I was thinking about a python file that contains the constants that I import to clean up the code. The code is admittedly messy too, and requires some moving around and making it more readable, especially the string concatenations.